README.md

```diff
@@ -83,19 +83,24 @@ var obj = {
 }
 ```
 
--- YOUR ANSWER HERE --
+ES5, no, obj.add(num1, num2)
```

Comment:
This is also ES6. See [Method Properties](http://es6-features.org/#MethodProperties)